### PR TITLE
Update ingest-data-streaming.md

### PR DESCRIPTION
**Pull request by TzviaGT**

### CODE REVIEW

**Comment by PRMerger7** [comment]
@TzviaGT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
Hi @TzviaGT - Contributors who work in the Product Manager role should open pull requests in the private repository for advanced quality validations and staging (you also can sign off directly in the private repo). To edit an article in the private repo from within the documentation, after you select **Edit** on an article page,  add “-pr” to repo name in the URL. We’ll leave this PR for review in the public repo, but would you make future article updates in the private repo?  Thanks!

File: data-explorer/ingest-data-streaming.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ There are two supported streaming ingestion types:
 
 ## Limitations
 
-* [Database cursors](kusto/management/databasecursor.md) aren't supported for database if the database itself or any of its tables have [streaming ingestion policy](kusto/management/streamingingestionpolicy.md) defined and enabled.
+* [Database cursors](kusto/management/databasecursor.md) aren't supported for database if the database itself or any of its tables have [streaming ingestion policy](kusto/management/streamingingestionpolicy.md) defined and enabled. (therefore no support for continuous export, update policy is limited to a query of the source table only)
 * [Data mapping](kusto/management/mappings.md) must be [pre-created](kusto/management/create-ingestion-mapping-command.md) for use in streaming ingestion. Individual streaming ingestion requests don't accommodate inline data mappings.
```
Comment (YaelSchuster) [inline data-explorer/ingest-data-streaming.md:70]:
```suggestion
* [Database cursors](kusto/management/databasecursor.md) aren't supported for a database if the database itself or any of its tables have [streaming ingestion policy](kusto/management/streamingingestionpolicy.md) defined and enabled. In this case there is no support for continuous export, and the update policy is limited to a query of the source table only.
* [Data mapping](kusto/management/mappings.md) must be [pre-created](kusto/management/create-ingestion-mapping-command.md) for use in streaming ingestion. Individual streaming ingestion requests don't accommodate inline data mappings.
```

**Comment by PRMerger14** [comment]
@orspod : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by orspod** [comment]
@TzviaGT - I committed the changes in the docs and signed off

**Pull request merged**